ci-operator/config/openshift/ironic-image/openshift-ironic-image-master__prevalidation.yaml

```diff
@@ -47,10 +47,18 @@ tests:
 - as: e2e-metal-ipi-prevalidation
   steps:
     cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG:
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN
     workflow: baremetalds-e2e
 - as: e2e-metal-ipi-virtualmedia-prevalidation
   steps:
     cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG:
+        IP_STACK=v4
+        NETWORK_TYPE=OpenShiftSDN
```

Comment:
There is no need giving environment variables for virtualmedia. It uses it's own https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml.